webrtc.html

```diff
@@ -1891,10 +1891,13 @@
               </dd>
               <dt><code>setConfiguration</code></dt>
```

Comment:
This whole section should probably point to JSEP somewhere since it goes into more detail.